composer.json

```
@@ -2,6 +2,8 @@
     "name": "storekeeper/magento2-plugin",
     "description": "Storekeeper plugin",
     "require": {
+        "php": ">=7.4|>=8.1",
```

there is reason we don't support 8 ?
```suggestion
        "php": "^7.4|^8",
```

---

@ikulis Well, I haven't tested fully on 8.1, and it is clear that this module was not developed under 8.x. But turned out that our test env are working under 8.1, so looks like we are good. Current magento 2.4.5-p1 support both 7.4 and 8.1, so we will go on '^7.4|^8.1' approach?

---

ok